CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## Fleet 4.12.1 (Apr 4, 2022)
+
+* Fix login error for non-SSO users when Fleet is deployed with a MySQL read replica.
```

Comment:
```suggestion
* Fix a bug in which a user could not log in with basic authentication. This only affects Fleet deployments that use a [MySQL read replica](https://fleetdm.com/docs/deploying/configuration#my-sql).
```

Reply:
Suggested tweak to the language that...
- Emphasizes the patch release fixes a "bug"
- Calls out what kind of Fleet deployments (those that use a read replica) are affected by the bug